src/main/java/se/michaelthelin/spotify/SpotifyHttpManager.java

```diff
@@ -81,9 +86,24 @@ public SpotifyHttpManager(Builder builder) {
         ? Timeout.ofMilliseconds(builder.connectTimeout)
         : ConnectionConfig.DEFAULT.getConnectTimeout())
       .build();
-    BasicHttpClientConnectionManager connectionManager = new BasicHttpClientConnectionManager();
-    connectionManager.setConnectionConfig(connectionConfig);
-    RequestConfig requestConfig = RequestConfig
+
+      if (usePooledConnectionManager)
+      {
+        connectionManager = new PoolingHttpClientConnectionManager();
+        SpotifyApi.LOGGER.log(
+          Level.FINE,
+          "SpotifyHttpManager - using PoolingHttpClientConnectionManager.");
```

Comment:
```suggestion
          "Using PoolingHttpClientConnectionManager");
```

Same below. The logger should add the class name as prefix already, I think.

Reply:
OK.